shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/context/CommonBeans.java

```diff
@@ -149,13 +149,6 @@ MetaStoreEventDeserializer metaStoreEventSerDe(ObjectMapper objectMapper) {
     return new JsonMetaStoreEventDeserializer(objectMapper);
   }
 
-  @Bean
-  ApiarySqsMessageDeserializer sqsMessageSerDe(
```

Comment:
There is also a bean for ObjectMapper in this class, can that be removed as well ?

Reply:
Good catch, `ObjectMapper` can be removed, as can the `MetaStoreEventDeserializer` bean

Reply:
Done